### PR TITLE
feat: Add interactive cherry-pick tool

### PR DESCRIPTION
**Pull request by joeyparrish**

### CODE REVIEW

File: interactive-cherry-pick/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Installation
+
+sudo ln -s $(pwd)/interactive-cherry-pick.sh /usr/local/bin/interactive-cherry-pick
+
+
+## Usage
+
+```sh
+git checkout v3.3.x  # release branch
+# "v3.3.1-master" is the last release tag on the master branch, so it will
```
Comment (theodab) [inline interactive-cherry-pick/README.md]: I think I get what you mean here, but the wording is odd. On first reading I thought this was, like, a declarative statement: "v3.3.1 is the last release tag". And I was like "this comment will no longer be accurate when we release v3.3.2!"
But no, I think you mean that that should be provided with the last release tag.

How about this wording instead?
```
For the first argument, you use the last release tag on the master branch. For example,
"v3.3.1-master". For the second argument, provide the name of the remote, such as "upstream". The third argument is the name of the main branch, such as "master".
```

**Review by theodab — CHANGES_REQUESTED** [review]

File: interactive-cherry-pick/README.md (hunk 1 above)
Comment (joeyparrish) [reply interactive-cherry-pick/README.md]: Hrm... Yeah, I see how this is confusing.  Let me put the explanation in prose before the shell script instead of sticking comments in it.  I'll also use some of your wording.

Comment (joeyparrish) [reply interactive-cherry-pick/README.md]: Please take a look at my latest push.  Is this clearer?

**Review by theodab — APPROVED** [review]
